packages/app/src/cli/commands/app/function/run.ts

```diff
@@ -42,10 +42,43 @@ export default class FunctionRun extends Command {
       path: flags.path,
       userProvidedConfigName: flags.config,
       callback: async (_app, ourFunction) => {
+        let functionExport = '_start'
+
+        if (flags.export !== undefined) {
+          this.log('Using export ${flags.export} from the --export flag.')
```

Comment:
We have to use the backticks for this to be a template string.

```suggestion
          this.log(`Using export ${flags.export} from the --export flag.`)
```

Reply:
Also I'm not sure what `this.log` is referring to here. Do you mean for this to be a debug log?

Reply:
Hm, yeah these should all be shifted to debug